app/sqaws.py

```diff
@@ -177,6 +294,18 @@ def set_tag(region_name: str, instance_id: str, tag_name: str, tag_value: str, d
         print(f'Response from create_tags: {str(response)}')
 
 
+# Set a tag on an instance or a volume (combined) - does this work?
+def set_tag(region_name: str, type_ec2: str, id_name: str, tag_name: str, tag_value: str, dryrun: bool) -> None:
+    ec2 = boto3.client('ec2', region_name=region_name)
+    print(f'Setting tag {tag_value} on {type_ec2}: {id_name} in region {region_name}')
```

Comment:
(fun fact) the id for volumes start with `v-` and instances start with `i-` so I think this comment would actually be unambiguous without the "on volume/instance" part - I don't have strong feelings about it though